sdk/signalr/Microsoft.Azure.WebJobs.Extensions.SignalRService/binding-library/java/README.md

```diff
@@ -0,0 +1,51 @@
+# SignalR Service Library for Azure Java Functions
```

Comment:
I don't think we would want to host Java code here. I'm not sure we would even be able to publish it from this repo.
@AlexGhiondea @weshaggard Should we consider moving this to the Java repo?

Reply:
I agree we shouldn't put java code here. Do you guys have a java SDK, or is this that? In either case as Josh calls out we cannot build or publish any java code from this repo.

Reply:
The java code hasn't been updated for a long time. I think I can just remove it. If needed later, I will move it to Java repo.